Dockerfile

```diff
@@ -14,6 +14,12 @@ RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
     pip install --no-cache-dir -r requirements/dev-requirements.txt && \
     # install mlflow in editable form
     pip install --no-cache-dir -e . && \
+    groupadd -g 10001 mlflow && \
+    useradd -u 10001 -g 10001 -m -d /home/mlflow mlflow
```

Comment:
This won't work on Windows, right? 
Doesn't the user need to be configured and created via `net user` command to be recognizable as a primary user so that the container group privileges can be applied and the user registered? 

https://docs.docker.com/engine/reference/builder/#user